fuzzy_file_nav.py

```diff
@@ -192,6 +192,8 @@ def on_query_context(self, view, key, operator, operand, match_all):
                 "fuzzy_bookmarks_load", "fuzzy_get_cwd", "fuzzy_cwv"
             ]:
                 return active
+            elif key == "fuzzy_go_to_parent_dir":
```

Comment:
Sorry! Just realized there was a mistake in functionality here. It was navigating up a directory on every backspace. This change makes it so that it only navigates up if the input is empty

Reply:
Good catch. I haven't tested functionality yet, but that is, of course, on my list.

Changes like this require more cautious evaluation. I think I noted  that I particularly wanted to make sure this feature behaves without issues, and you've just stumbled on why 🙂.

Reply:
Totally agree! There's no rush or pressure on my end to get the changes merged since I can already use them locally. My goal was just to add some options that maybe other users would find helpful